### DISCUSSION TITLE
Not all functions exported in module

### DISCUSSION THREAD

**Discussion by bftblomster** [microsoft/winget-cli#2088] [PowerShell]
https://github.com/microsoft/winget-cli/blob/cf85374d37cc9391e22bff62aac9b62b524d4398/tools/PowerShell/Microsoft.WinGet.Client/src/Microsoft.WinGet.Client.psd1#L70

Is there a reason only these cmdlets are available? Do the other functions in Library not work yet?

**Comment by denelon** [comment]
I think we were just using that method essentially "one at a time" with cmdlets as we were building them out to test syntax. I don't remember exactly as it was quite a while back. We had several challenges related to the text output from winget.exe. That's why we're working on a bridge to the COM APIs so we can have support for PowerShell 5.1, 6, and 7.
